### ISSUE TITLE
How can I run multiple chains?

### ISSUE THREAD

**Issue by ghost** [pymc-devs/pymc3#77]
Hi,

I am fairly used to OpenBUGS and JAGS and want to begin using python with pymc for my bayesian estimation needs. I was wondering however how one runs multiple chains? In JAGS you simply add an option n.chains=2 or n.chains=4 but this doesn't seem to be the case with pymc. How should I proceed? I want to plot the traces of the different chains over one another to be able to see if mixing is good.

Sorry if this question was already answered, I used the search function but didn't find anything.

Oh, and merry Christmas!

**Comment by fonnesbeck** [comment]
PyMC only runs chains in serial, rather than in parallel. We are working on ways to parallelize the chains, but none of that is working yet. When one sample() call is finished, that comprises one chain; the next time sample() is run on the same MCMC object, it generates a new chain.

M = MCMC(my_model)
for i in 1:chains:
    M.sample(10000, 5000)

Summary plots and statistics are, by default, run on the last chain, though Matplot.plot() and M.stats() can be given an optional "chain" argument to pull out individual chains.

This is more of a question than an issue, so I am closing this. I will open another issue for allowing plots to output multiple chains.
